### PR TITLE
gad-3 : week 6 : changed name depthead to sectionIncharge

### PR DESCRIPTION
**Pull request by ujjawallele13**
### Brief description of what is fixed or changed

Changed the name of Depthead to SectionIncharge
Rest functionality is same as was in the previous PR  https://github.com/FusionIIIT/Fusion/pull/1088

incharge username
incharge extra info
electricity incharge entry
worker for incharge

### CODE REVIEW

**Comment by PranshuNayak** [comment]
@ujjawallele13 please attach the screenshot of changes reflected in the Django-admin portal after you have changed the title to 'section in charge' to ensure there are no conflicts. Also, what is the need to rename the table? Please make meaningful commit messages .

**Comment by PranshuNayak** [comment]
@akshatnema please make a guideline for commit messages. All commit messages must depict the scope of the commit like Database, UI, or API. @ujjawallele13 please follow the standard commit message guidelines whenever you push any changes. Also don't push so many commits in a single PR, whenever you push some change get it reviewed first , before doing more changes and if approved squash the commit messages into a single meaningful commit message.

File: FusionIIIT/Fusion/settings/development.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'fusionlab',
         'HOST': 'localhost',
-        'USER': 'fusion_admin',
-        'PASSWORD': 'hello123',
```
Comment (samay-rgb) [inline FusionIIIT/Fusion/settings/development.py:15]: @ujjawallele13 you don't need to add this file to your PRs. You can remove this file from this PR.

**Comment by PranshuNayak** [comment]
@ujjawallele13 please squash your commit messages to some meaningful commits. It's really hard to understand what changes have you made by reading your commits.

File: FusionIIIT/Fusion/urls.py [hunk 2]
```diff
@@ -22,14 +22,8 @@
...
 from django.contrib.auth import views as auth_views
 
-from rest_framework.authtoken.views import obtain_auth_token 
 
-
-
-urlpatterns = [
-
-    url(r'^api-token-auth/', obtain_auth_token, name='api_token_auth'),  
-
```
Comment (PranshuNayak) [inline FusionIIIT/Fusion/urls.py]: why have you removed the token? Please elaborate

Comment (ujjawallele13) [reply FusionIIIT/Fusion/urls.py]: I was using it at the time of api's testing. Well there is no problem in keeping it.

Comment (PranshuNayak) [reply FusionIIIT/Fusion/urls.py]: Whatever changes you are doing while testing don't push them, only push the necessary changes related to the PR

**Pull request closed without merging**